projects/ontimize-web-ngx/src/lib/layouts/form-layout/split-pane/o-form-layout-split-pane.component.html

```diff
@@ -0,0 +1,21 @@
+<div fxFill>
+  <div fxLayout="row" class="row-wrapper">
+    <div #mainWrapper fxFlex fxLayout="row" class="main-content">
+      <ng-content></ng-content>
+    </div>
+
+    <div #detailWrapper class="content-wrapper" mwlResizable [enableGhostResize]="true"
+      (resizeEnd)="onResizeEnd('width', $event, detailWrapper)">
+
+      <div class="resize-handle" mwlResizeHandle [resizeEdges]="{ left: true }">
+        <div class="resize-border">
+          <div></div>
+        </div>
+      </div>
+
+      <div class="secondary-content" fxLayout="column" fxLayoutAlign="center stretch" fxFlex>
```

Comment:
Can we rename css classes to something that starts with `o-` and makes reference to the current component?